lib/tapioca/commands/command.rb

```diff
@@ -21,11 +21,18 @@ def initialize
         @file_writer = T.let(FileWriter.new, Thor::Actions)
       end
 
-      sig { abstract.void }
-      def execute; end
+      sig { void }
```

Comment:
Should we mark it as `final` so we're sure commands can't break the encapsulation?